src/tests/excalidrawPackage.test.tsx

```diff
@@ -130,4 +130,66 @@ describe("<Excalidraw/>", () => {
       expect(textInput?.nodeName).toBe("SPAN");
     });
   });
+
+  describe("Test UIOptions prop", () => {
+    it('should not hide any UI element when the UIOptions prop is "undefined"', async () => {
+      const { container } = await render(<Excalidraw />);
+
+      expect(queryByTestId(container, "clear-canvas-button")).toBeTruthy();
+      expect(queryByTestId(container, "export-button")).toBeTruthy();
+      expect(queryByTestId(container, "load-button")).toBeTruthy();
+      expect(queryByTestId(container, "save-as-button")).toBeTruthy();
+      expect(queryByTestId(container, "save-button")).toBeTruthy();
+    });
+
+    it('should not hide any UI element when canvasActions is "undefined"', async () => {
+      const { container } = await render(<Excalidraw UIOptions={{}} />);
+
+      expect(queryByTestId(container, "clear-canvas-button")).toBeTruthy();
+      expect(queryByTestId(container, "export-button")).toBeTruthy();
+      expect(queryByTestId(container, "load-button")).toBeTruthy();
+      expect(queryByTestId(container, "save-as-button")).toBeTruthy();
+      expect(queryByTestId(container, "save-button")).toBeTruthy();
+    });
```

Comment:
might be better to just take snapshot of the canvas actions instead of asserting every item individually. And if more items get rendered in future snap will fail as well whereas this won't unless assertion.

Reply:
Awesome! Makes sense. I'm new to testing, forgive my mistakes! 😅

Reply:
No worries at all :)